llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -970,6 +970,9 @@ def FeatureFastUnalignedAccess
 def FeaturePostRAScheduler : SubtargetFeature<"use-postra-scheduler",
     "UsePostRAScheduler", "true", "Schedule again after register allocation">;
 
+def FeatureNoRAS : SubtargetFeature<"no-ras", "HasRAS", "false",
+                                    "Hasn't RAS (Return Address Stack)">;
+
```

Comment:
https://github.com/llvm/llvm-project/blob/8f7fdd94ef19af7b4905b316c253a78219a6038f/llvm/lib/Target/ARM/ARM.td#L418-L422

Not sure which one is better.

Reply:
Yeah, RAS can be ambiguity as RAS can be `Reliability, Availability, Serviceability`.
I will align it to ARM later.